### DISCUSSION TITLE
MacOS ARM 2.3.57.9 build broken

### DISCUSSION THREAD

**Discussion by Evanion** [supermerill/SuperSlicer#2207] [General]
I tried downloading the latest arm build for macos, twice, but macos can't start it, saying it's broken. Downloaded the regular mac build, and could start that.
Computer:
Model Name:	MacBook Pro
  Model Identifier:	MacBookPro18,1
  Chip:	Apple M1 Pro
  Total Number of Cores:	10 (8 performance and 2 efficiency)
  Memory:	32 GB
  System Firmware Version:	7429.61.2
  OS Loader Version:	7429.61.2

<img width="254" alt="Screenshot 2022-01-11 at 01 29 25" src="https://user-images.githubusercontent.com/319799/148860488-be98bb29-d35b-427f-8513-e8ba057bcb56.png">

**Comment by supermerill** [comment]
see #2162
